modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/FormURL.es.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import FormURL from '../../../src/main/resources/META-INF/resources/admin/js/util/FormURL.es';
+
+const formInstanceId = 1;
+
+describe('FormURL', () => {
+	it('gets a blank url', () => {
+		const formURL = new FormURL(1, false);
+		expect(formURL.create()).toBe('');
+	});
+
+	it('gets a valid restricted url', () => {
+		const formURL = new FormURL(1, true, true);
```

Comment:
@marcosapmf, can you please create local variables to store the parameters values and use these variables as parameters? It'll help the reader which doesn't know the parameters of `FormURL` previously. 😉